contactmps/static/javascript/secretballot.js

```diff
@@ -56,15 +58,23 @@ $(".follow-up-question-box .toggle-select-follow-up").click(function() {
   $(".follow-up-question-box .toggle-select-follow-up").removeClass("selected");
   $this.addClass("selected");
 
-  var q = $('.follow-up-question').text();
-  var a = $this.text();
+  var q = $('.follow-up-question').text().trim();
+  var a = $this.text().trim();
 
   $('.follow-up-question p').text('Thanks!');
   $('.follow-up-answer-box').hide();
 
-  ga('send', 'event', 'follow-up', 'answered', q);
+  // submit to server
+  jQuery.ajax('/api/v1/email/' + emailId + '/qa/', {
+    type: 'POST',
+    data: {
+      question: q,
+      answer: a,
+      sender_secret: senderSecret,
+    },
+  });
```

Comment:
it'd be nice to keep logging errors to the console unless that's considered very bad practise in production